docs/install-docker.md

```diff
@@ -1,40 +1,60 @@
 
-OpenSAFELY uses Docker to run your code in a reproducible, safe manner. 
+OpenSAFELY uses Docker to run your code in a reproducible, safe manner.
 This is most helpful for checking that you will be able to successfully run your code on the OpenSAFELY server on real data.
 If you only run your code locally using your own installation of R, say, then you won't know if the version of R (and the packages) installed on the server will run your code without errors or unexpected behaviours.
 See the [Testing Your Code section](actions-pipelines.md) for more details on how to test your code in practice.
 
-Unfortunately, Docker is happiest on Linux; on Windows and Mac OSX, installation can be a chore. 
+Unfortunately, Docker is happiest on Linux; on Windows and Mac OSX, installation can be a chore.
 These notes should help.
 
 ## Installation
 
-Windows and Macs have different installation processes. 
-Regardless of machine, you will have to install Docker and make an account on the [Docker Website](https://docs.docker.com/).
-
-There are two flavours you can install, *Desktop* and *Toolbox*. 
-Docker Desktop is preferred over Docker Toolbox. 
+Windows and Macs have different installation processes.
+Regardless of machine, you will have to install Docker Desktop and make an account on the [Docker Website](https://docs.docker.com/).
 
 ### Windows
 
-Docker Desktop in Windows offers native support via Hyper-V containers, and so is preferred.
+Follow the instructions from the Docker website.
+Windows 10 or later is required.
+It is best to install using the default settings.
 
-To install Docker Desktop on Windows 10 64-bit Pro, Enterprise, or Education build 15063 or later (i.e., most university or institution managed machines), [follow these installation instructions](https://docs.docker.com/docker-for-windows/install/).
-To install Docker Desktop on Windows Home [follow these installation instructions](https://docs.docker.com/docker-for-windows/install-windows-home/).
+!!! note
+    We previously recommended Docker Desktop with the Hyper-V backend.
 
-Windows users who log into an Active Directory domain (i.e., a network login) may find they lack permissions to start Docker correctly. 
-If so, [follow these instructions](https://github.com/docker/for-win/issues/785#issuecomment-344805180).
-It is best to install using the default settings.
+    Docker Desktop's recommended configuration is now the WSL2 backend.
+    We have not thoroughly tested OpenSAFELY with WSL2,
+    but there is no reason why WSL2 should be incompatible with OpenSAFELY's tooling.
 
-You may be asked to enable the Hyper-V and Containers features, which you should do.
-You can do this by [following these instructions](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v). 
+    You only need to choose either Hyper-V or WSL2.
+    It is fine to continue to use the Hyper-V backend.
```

Comment:
As a reader, this makes me nervous. The Docker people say "WSL2". The OpenSAFELY people say "WSL2, but we haven't tested it, and Hyper-V is fine".

Could we (as OpenSAFELY people) commit to WSL2, and keep a very close eye out for issues?

Reply:
> Could we (as OpenSAFELY people) commit to WSL2, and keep a very close eye out for issues?

This is probably the right thing to do seeing as that's where Docker looks to be headed.

I'd be happier if a friendly Windows user could test WSL2 so that we know it does work. (I don't expect that it won't. But confirmation would be great :heavy_check_mark:)

I don't have a native Windows install to test with, unfortunately.